hack/install_devconsole/README.md

```diff
@@ -1,25 +1,39 @@
-##Non-admin user viewing the console with developer console perspective
+## Developer console for non-admin users
 
-Refers [https://jira.coreos.com/browse/ODC-347](https://jira.coreos.com/browse/ODC-347)
+You can use `consoledeveloper.sh` script available in this directory
+to enable developer console for a non-admin user.
 
```

Comment:
We should also add to the list of pre-conditions that the script should be run to install developer perspective on an OCP4 cluster.

Reply:
@Avni-Sharma Did we miss to specify some pre-condition?

Reply:
Cloning this repo locally, also as @rgarg1 suggested we can specify the min required version of oc

Reply:
@Avni-Sharma I will make the changes and update the PR.

Reply:
They see README and script after cloning the repo. So not required.